src/tiled/propertiesdock.cpp

```diff
@@ -91,6 +91,11 @@ PropertiesDock::PropertiesDock(QWidget *parent)
 
     setWidget(widget);
 
+    contextMenu = new QMenu(mPropertyBrowser);
```

Comment:
Please allocate this on the stack only where needed (in `showContextMenu`). Just put it like that:
```c++
    QMenu contextMenu(mPropertyBrowser);
```